### ISSUE TITLE
npx fails

### ISSUE THREAD

**Issue by TheJaredWilcurt** [amzn/style-dictionary#212]
```cmd
C:\style-dic> node -v
v10.13.0
C:\style-dic> npm -v
6.4.1
C:\style-dic> npx -v
6.4.1
C:\style-dic> npm init -y
C:\style-dic> npm install --save-dev style-dictionary
C:\style-dic> npx style-dictionary init basic
Copying starter files...

ENOENT: no such file or directory, lstat 'C:\style-dic\node_modules\style-dictionary\example\basic'
```

Related:

* [Why NPM global installs are an anti-pattern](https://projectsunday.github.io/docs/npm-global)

**Comment by TheJaredWilcurt** [comment]
Even if I clone the repo down directly the `example` folder is actually named `examples`. Renaming it still doesn't work. you're stuck manually downloading the files, which there is no direct link to, just to the folder on github. So you have to download or clone the whole repo just to get the `basic` example.

This is a realllllllly rough "getting started" guide.

**Comment by dbanksdesign** [comment]
v2.6.1 is now released, you should be able to run `npx style-dictionary init basic` now

**Comment by dbanksdesign** [comment]
I was a bit too quick, there was another issue, but I fixed it, v2.6.2 is now released and I have tested the npx command works now.

**Comment by TheJaredWilcurt** [comment]
Works as expected now
